packages/wonder-blocks-button/components/button.js

```diff
@@ -61,6 +61,8 @@ export type SharedProps = {
      */
     testId?: string,
 
+    tag?: React.ComponentType<*>,
```

Comment:
Hmmm... not a huge fan of this. The only time it should be used is for client-side nav so letting anything else through doesn't seem good.